Workbooks/Workloads/Collection logs/Collection logs.workbook

```diff
@@ -246,7 +246,7 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "Operation\n| where Computer =~ '{OnboardedComputers}'\n| where Detail !contains \"mssql: Incorrect syntax near ','.\"\n| where OperationStatus == \"Error\" or OperationStatus == \"Warning\" or \"{LogsFilter}\" == \"All Logs\"\n| extend jk=1\n| join kind=leftouter (\n    datatable (CommonErrors:dynamic)[dynamic({CommonErrors})]\n    | mv-expand CommonErrors\n    | extend jk=1) on jk\n| extend ContainsCommonError = Detail contains CommonErrors.key\n| summarize by OperationStatus, Time = TimeGenerated, Message = Detail, Action = iff(ContainsCommonError, tostring(CommonErrors.action), \"\"), Description = iff(ContainsCommonError, tostring(CommonErrors.description), \"\")\n| order by Time, Message, Action desc\n| extend isValidRow = iff(Message == prev(Message) and Time == prev(Time), iff(Action != prev(Action) and isnotempty(Action), true, false), true)\n| where isValidRow\n| take 500",
+              "query": "Operation\n| where Computer =~ '{OnboardedComputers}'\n| where OperationStatus == \"Error\" or OperationStatus == \"Warning\" or \"{LogsFilter}\" == \"All Logs\"\n| union (WorkloadDiagnosticLogs\n    | where Computer =~ '{OnboardedComputers}'\n    | where Status == \"Error\" or Status == \"Warning\" or \"{LogsFilter}\" == \"All Logs\"\n    | project TimeGenerated, Detail=Message, OperationStatus=Status)\n| extend jk=1\n| join kind=leftouter (\n    datatable (CommonErrors:dynamic)[dynamic({CommonErrors})]\n    | mv-expand CommonErrors\n    | extend jk=1) on jk\n| extend ContainsCommonError = Detail contains CommonErrors.key\n| summarize by OperationStatus, Time = TimeGenerated, Message = Detail, Action = iff(ContainsCommonError, tostring(CommonErrors.action), \"\"), Description = iff(ContainsCommonError, tostring(CommonErrors.description), \"\")\n| order by Time, Message, Action desc\n| extend isValidRow = iff(Message == prev(Message) and Time == prev(Time), iff(Action != prev(Action) and isnotempty(Action), true, false), true)\n| where isValidRow\n| take 500",

```

Comment:
need to rethink this actually, as it'll error if WorkloadDiagnosticLogs doesn't exist. I might just have to have two visualizations that show one or the other based on a required version. Then afterwards in a later change, remove the query against Operation and only have it again WorkloadDiagnosticLogs

Reply:
metadata for WorkloadDiagnosticLogs table is being rolled out still - it should reach all regions Friday or Monday, at that time can query for it from all regions.